### PR TITLE
Use TypeScript 1.5 module

### PR DESCRIPTION
**Pull request by blakeembrey**
Relies on https://github.com/tkellen/js-interpret/pull/32

### CODE REVIEW

**Comment by tkellen** [comment]
Can you ping me when this is ready for review?

**Comment by blakeembrey** [comment]
@tkellen It's working now! Sorry, got caught up with lots of other work. I had to use the latest master since the latest stable doesn't seem to support TSX yet. Current `tsconfig.json` lookup is based on `process.cwd()`, so it's in the root right now - I can figure out how to move it if that's an issue.

**Comment by tkellen** [comment]
Thanks @blakeembrey! I've just added you as a collaborator on this and interpret.

File: test/index.js [hunk 1]
```diff
@@ -167,6 +167,10 @@ describe('rechoir', function () {
       this.timeout(5000);
       rechoir.prepare(extensions, './test/fixtures/test.ts');
       expect(require('./fixtures/test.ts')).to.deep.equal(expected);
```
Comment (tkellen) [inline test/index.js]: missing a closing }); here

**Comment by tkellen** [comment]
looking in `process.cwd()` for `tsconfig.json` isn't going to work if you execute gulp from a child directory of your project. You should check out `findup-sync`, or just use liftoff for whatever needs to find tsconfig!

File: test/index.js (hunk 1 above)
Comment (blakeembrey) [inline test/index.js]: Whoops! I was half asleep by this point.

**Comment by blakeembrey** [comment]
> looking in process.cwd() for tsconfig.json isn't going to work if you execute gulp from a child directory of your project

That's actually what it does already. It just starts from the cwd and moves upward.

**Comment by blakeembrey** [comment]
@tkellen There's a couple of failures still but I'll try investigating it later. Looks like things work though :smile:

**Comment by blakeembrey** [comment]
@tkellen All done! The only failures are still `typescript-register` from before. I ended up having to do a release on `typescript-node` to support the `VERSION` export - which lead me to writing a really fun bash script with sed to replace the version on pre commit :smile:

**Comment by blakeembrey** [comment]
I also moved `tsconfig.json` into the fixtures directory and just `chdir` into it before the tests run.

**Comment by tkellen** [comment]
:+1: thanks @blakeembrey!

**Pull request merged**